### PR TITLE
Change to use Edeng notation for scout_margin

### PR DESCRIPTION
**Pull request by neuhausj**
In order to have the same notation across all forks for scout_margin and ease integration in the Telegram bot.
It uses 2 parameters:
* use_margin (yes/no)
* scout_margin (expected coin gain in % after fee deduction)

### CODE REVIEW

**Comment by SergeyKubrak** [comment]
Hi! 
Looks like we can clean up `Config` and remove `Config.RATIO_CALC_DEFAULT` and `Config.RATIO_CALC_SCOUT_MARGIN` attributes.  Also check `crypto_trading.py:41` it will raises exception as you're not defining `Config.RATIO_CALC` anymore.

**Comment by neuhausj** [comment]
Thanks ! I just added the 2 changes you proposed ;)

File: binance_trade_bot/config.py [hunk 1]
```diff
@@ -173,3 +157,6 @@ def __init__(self):
 
         allow_coin_merge = os.environ.get("ALLOW_COIN_MERGE") or config.get(USER_CFG_SECTION, "allow_coin_merge")
         self.ALLOW_COIN_MERGE = str(allow_coin_merge).lower() == 'true'
+
+        self.USE_MARGIN = os.environ.get("USE_MARGIN") or config.get(USER_CFG_SECTION, "use_margin")
```
Comment (tntwist) [inline binance_trade_bot/config.py:161]: We already have some bool flags in the config values which use "true" and "false". please change it to "true"/"false" notation. You could copy the code from the "allow_coin_merge" setting.

File: binance_trade_bot/auto_trader.py [hunk 2]
```diff
@@ -147,19 +147,19 @@ def _get_ratios(self, coin: Coin, coin_price, excluded_coins: List[Coin] = []):
...
+            # Fees
             from_fee = self.manager.get_fee(pair.from_coin, self.config.BRIDGE, True)
-            to_fee =  self.manager.get_fee(pair.to_coin, self.config.BRIDGE, False)
-
-            if self.config.RATIO_CALC == self.config.RATIO_CALC_DEFAULT:
-                transaction_fee = from_fee + to_fee
+            to_fee = self.manager.get_fee(pair.to_coin, self.config.BRIDGE, False)
+            transaction_fee = from_fee + to_fee - from_fee * to_fee
 
+            if self.config.USE_MARGIN == "yes":
```
Comment (tntwist) [inline binance_trade_bot/auto_trader.py:155]: This would need to be changed too when we use a bool value.

**Comment by tntwist** [comment]
Hi there. Thanks for the clean up. ❤️

**Pull request merged**